application/src/main/java/org/thingsboard/server/service/entitiy/DefaultTbNotificationEntityService.java

```diff
@@ -202,6 +203,21 @@ public void notifyDeleteAlarm(Alarm alarm, SecurityUser user, List<EdgeId> relat
         sendAlarmDeleteNotificationMsg(alarm, relatedEdgeIds);
     }
 
+    @Override
+    public void notifySaveCustomer(Customer customer, ActionType actionType, SecurityUser user, Object... additionalInfo) {
```

Comment:
Name should be like other methods `notifyCreateOrUpdate`...
The asset method has the same logic, maybe should create method `notifyCreateOrUpdateEntity`?

Reply:
Ok